solr/core/src/test/org/apache/solr/cloud/DeleteShardTest.java

```diff
@@ -79,14 +80,19 @@ public void test() throws Exception {
     waitForState("Expected 'shard1' to be removed", collection, (n, c) -> {
       return c.getSlice("shard1") == null;
     });
+    
+    // verify shard1 znodes are deleted
+    assertShardZnodesDeleted(collection, "shard1");
 
     // Can delete a shard under construction
     setSliceState(collection, "shard2", Slice.State.CONSTRUCTION);
     CollectionAdminRequest.deleteShard(collection, "shard2").process(cluster.getSolrClient());
     waitForState("Expected 'shard2' to be removed", collection, (n, c) -> {
       return c.getSlice("shard2") == null;
     });
-
+    
+    // verify shard2 znodes are deleted
+    assertShardZnodesDeleted(collection, "shard");
```

Comment:
Shouldn't this be:

`assertShardZnodesDeleted(collection, "shard2");`

Reply:
Good catch, thanks!